### ISSUE TITLE
Missing hover text on many places

### ISSUE THREAD

**Issue by darrenge** [Microsoft/pxt#87]
I believe Hover text is used by accessibility tools so to help with accessibility and just overall help, we need to add hover text. There are a few (undo, show code, embed) that have hover text but most don't.

Things that need update text:
- [x] More menu and all items in the More menu
- [x] ? menu and all items in the ? menu
- [x] Getting Started Pane expand / collapse button
- [ ] Getting Started Pane show code
- [x] Getting Started Pane Run
- [x] Getting Started Pane Download
